docs/user-guide/topics/metrics_store.md

```diff
@@ -0,0 +1,54 @@
+---
+template: main.html
+---
+
+# Metrics store
+
+One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. This metrics store, based on BadgerDB is not suitable for production use. It is suitable only for a single instance of Iter8.
+
+To support production use cases, the metrics store can be replaced with another distributed database such as Redis. Currently Iter8 supports:
+
+- BadgerDB
+- Redis
```

Comment:
I think this section should first mention the supported databases, and then mention which are suitable for production use cases.

For example:

One of Iter8's key advantages is that it incorporates its own metrics store, simplifying the set up and execution of A/B/n and performance tests. Iter8 currently supports the following databases:

- BadgerDB
- Redis

Note that BadgerDB is not suitable for production use and is only suitable for a single instance of Iter8. Support for other databases are in the works. See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.